src/encoded/static/components/experiment_series.js

```diff
@@ -520,6 +529,9 @@ class ExperimentSeriesComponent extends React.Component {
         let internalTags = [];
         const allRows = [];
         const { viewableDatasets } = this.state;
+        const handleAnalysisRFAChange = (e) => {
+            this.setState({ selectedAnalysisRFA: e.target.value });
+        };
```

Comment:
`setState` inside render() in a class component shouldn’t happen, as you can easily have infinite render loops. Should make this function its own method. You might find examples of this elsewhere, but they shouldn’t have happened.

Reply:
Good point. Thank you! I will make it a method and use `this. handleAnalysisRFAChange` upon changes.